src/prefect/workers/base.py

```diff
@@ -51,6 +51,7 @@
     get_worker_logger,
 )
 from prefect.plugins import load_prefect_collections
+from prefect.server.schemas.states import State
```

Comment:
We want to avoid importing from `prefect.server` here because it greatly increases the import time.
```suggestion
from prefect.states import Cancelled
```

Reply:
Updated [`dc8c0da` (#17537)](https://github.com/PrefectHQ/prefect/pull/17537/commits/dc8c0dae081032050f9b40bce39959e9efb167c6)